### PR TITLE
Java 11 compatible release

### LINKED ISSUE #18: Version of annotation-indexer is really old
Current version is 1.12 vs 1.4 currently used. Main apparent advantage in upgrading is elimination of the javac compiler annotation processor warning:

`[WARNING] warning: Supported source version 'RELEASE_6' from annotation processor 'org.jvnet.hudson.annotation_indexer.AnnotationProcessorImpl' less than -source '1.8'`

### PR DESCRIPTION
**Pull request by davido**
Closed #18.

In #19 annotation-indexer was updated to 1.12, but repository was
not changed to repo.jenkins-ci.org.

Also bump version from 1.19-SNAPSHOT to 1.19 to prepare next
release iteration.

### CODE REVIEW

File: injector/pom.xml [hunk 1]
```diff
@@ -162,8 +168,28 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-tree</artifactId>
+      <version>9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-analysis</artifactId>
+      <version>9.0</version>
```
Comment (bondolo) [inline injector/pom.xml:192]: Won't all of these come in as dependencies of the base asm dependency?

Comment (davido) [reply injector/pom.xml]: Asm's pom.xml doesn't include transitive dependencies: [1].
However, it seems that only `asm-commons` is actually used.
I removed all other transitive dependencies. Done.

[1] https://repo1.maven.org/maven2/org/ow2/asm/asm/9.0/asm-9.0.pom

Comment (lucamilanesio) [reply injector/pom.xml]: Thanks @davido, the fix is much appreciated 👍

**Comment by timja** [comment]
@irobertson could you take a look please?

File: annotation/pom.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>com.infradna.tool</groupId>
     <artifactId>bridge-method-injector-parent</artifactId>
-    <version>1.19-SNAPSHOT</version>
+    <version>1.19</version>
```
Comment (jglick) [inline annotation/pom.xml:6]: Not sure what is going on here.

Comment (jglick) [reply annotation/pom.xml:6]: No such edit should ever be made as part of a functional PR. It is up to maintainers to prepare releases and any associated metadata edits. (In the case of this repository, using `maven-release-plugin`.)

Comment (davido) [reply annotation/pom.xml:6]: > It is up to maintainers to prepare releases and any associated metadata edits.

But they haven't done it for years... Let me revert than change.

Comment (davido) [reply annotation/pom.xml:6]: This project was unmaintained even before @kohsuke disappeared.

**Comment by kohsuke** [comment]
I merged 806815c390da3d028b01cf08883e5aea769261ec excluding one commit that did a release version change.

**Comment by kohsuke** [comment]
This change shipped in 1.21.

**Pull request closed without merging**